services/libssh/consts.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+
+SERVER_SIG = "SSH-2.0-libssh_0.7.4"
+
+CVE_SSH_PORT = 2200
```

Comment:
Why is this default port different from the default value of the argument? If the port is not specified, which will be used?

Reply:
Actually the usage of this const is on to run the server not through honeycomb, but directly (check main func).
I've changed this back to 2222.